sgqlc/codegen/schema.py

```diff
@@ -129,6 +130,10 @@ def __init__(self, schema_name, schema, writer, docstrings):
         self.mutation_type = self.get_path('mutationType', 'name')
         self.subscription_type = self.get_path('subscriptionType', 'name')
         self.directives = schema.get('directives', [])
+        self.builtin_types = ('Int', 'Float', 'String', 'Boolean', 'ID')
+        self.datetime_types = () \
+            if disable_datetime_types else('DateTime', 'Date', 'Time')
+        self.relay_types = ('Node', 'PageInfo')
```

Comment:
while I see the rationale behind this, it's somehow expected to have such issues in the future as well... could you please convert your patch to change the boolean and tuples to an optional mapping of types to package?

something like:

```py
builtin_types_import = 'sgqlc.types'
builtin_scalar_imports = {
    k: builtin_types_import for k in ('Int', 'Float', 'String', 'Boolean', 'ID')
}
datetime_scalar_imports = {
    k: 'sgqlc.types.datetime' for k in ('DateTime', 'Date', 'Time')
}
relay_imports = {
    k: 'sgqlc.types.relay' for k in ('Node', 'PageInfo')
}

default_type_imports = { **builtin_scalar_imports, **datetime_scalar_imports, **relay_imports }

class CodeGen:
def __init__(self, schema_name, schema, writer, docstrings, type_imports=default_type_imports):
     ...
     self.type_imports = type_imports

...
    def analyze(self):
        self.imports = set()
        for t in self.types:
            name = t['name']
            try:
                self.imports.add(self.type_imports[name])
            except KeyError:
                pass
        try:
            self.imports.remove(builtin_types_import)
        except KeyError:
            pass

    def write_header(self):
        self.writer('import sgqlc.types\n')
        for imp in sorted(self.imports):
            self.writer('import %s\n' % imp)
        ...

    def write_type_scalar(self, t):
        name = t['name']
        imp = self.type_imports.get(name):
        if imp:
            self.writer('%s = %s.%s' % (name, imp, name))

```

NOTE: I did not test this, could you please try to implement something along these lines?